src/network/models/example-models.ts

```diff
@@ -0,0 +1,22 @@
+//
+//
+//
+//
+// THIS IS NOT FUNCTIONAL, JUST FOR EXAMPLE PURPOSES, REMOVE WHEN NOT NEEDED
+//
+//
+//
+export type ExampleRequest = {
+  pet: {
+    name: string;
+    age?: number;
+  };
+};
+
+export type ExampleResponse = {
+  pet: {
+    id: string;
+    name: string;
+    age?: number;
+  };
+};
```

Comment:
I saw a few days ago this library [zod](https://zod.dev/?id=basic-usage). Will depend on the use case but could be useful.

Reply:
that is for validations, not sure if it helps you define ts models for the endpoints